### DISCUSSION TITLE
What does a light blue Device Name signify?

### DISCUSSION THREAD

**Discussion by Two38** [lostindark/DriverStoreExplorer#257] [General]
well, the title says it all. Am I somehow missing where its documented what that means? Appreciate any (decent, non-incendiary) reply.

**Comment by JonnyTech** [comment]
Can you share a screenshot?

**Reply by Two38** [reply]
you can see this on the demo screenshot here
 https://github.com/lostindark/DriverStoreExplorer/blob/master/Screenshots/Screenshot.png 
(Logitech Gaming).

**Reply by JonnyTech** [reply]
#231
